tests/manage/mcg/test_verify_noobaa_status.py

```diff
@@ -20,4 +24,16 @@ def test_verify_noobaa_status_cli(mcg_obj_session):
     for line in status.split("\n"):
         if "Not Found" in line:
             assert "Optional" in line, f"Error in noobaa status output- {line}"
+        if float(config.ENV_DATA["ocs_version"]) > 4.7 and "Noobaa-db" in line:
```

Comment:
you can use semantic method
``` 
ocs_version_semantic = version.get_semantic_ocs_version_from_config()
if ocs_version_semantic >= version.VERSION_4_7
```

Reply:
Adding on top of Oded's suggestion - 
```suggestion
        if float(config.ENV_DATA["ocs_version"]) > 4.7 and "noobaa-db" in line.lower():
```

Reply:
Thanks both. Ack.